build.sbt

```diff
@@ -712,48 +545,14 @@ credentials ++= (
   )
 ).toSeq
 
-val jvmProjects = Seq(
-  "numbers",
-  "core",
-  "generic",
-  "genericExtras",
-  "shapes",
-  "refined",
-  "parser",
-  "scodec",
-  "tests",
-  "jawn",
-  "jackson"
-) ++ (
-  if (sys.props("java.specification.version") == "1.8") Seq("java8", "optics") else Nil
-)
-
-val jvmTestProjects = Seq(
-  "numbers",
-  "tests"
-) ++ (
-  if (sys.props("java.specification.version") == "1.8") Seq("java8", "optics") else Nil
-)
-
-val jsProjects = Seq(
-  "numbersJS",
-  "coreJS",
-  "genericJS",
-  "genericExtrasJS",
-  "shapesJS",
-  "opticsJS",
-  "parserJS",
-  "refinedJS",
-  "scalajs",
-  "scodecJS",
-  "testsJS"
-)
+/* Only run optics tests on Java 8 since Scala 2.11 on Java 7 breaks. */
```

Comment:
Documented here what I believe is the reason to only run optics tests on JVM 8 and made sure optics is built regardless of the JVM version.